astroid/brain/brain_typing.py

```diff
@@ -240,16 +240,17 @@ def _forbid_class_getitem_access(node: ClassDef) -> None:
     def full_raiser(origin_func, attr, *args, **kwargs):
         """
         Raises an AttributeInferenceError in case of access to __class_getitem__ method.
-        Otherwise just call origin_func.
+        Otherwise, just call origin_func.
         """
         if attr == "__class_getitem__":
             raise AttributeInferenceError("__class_getitem__ access is not allowed")
         return origin_func(attr, *args, **kwargs)
 
     try:
         node.getattr("__class_getitem__")
-        # If we are here, then we are sure to modify object that do have __class_getitem__ method (which origin is one the
-        # protocol defined in collections module) whereas the typing module consider it should not
+        # If we are here, then we are sure to modify object that do have
+        # __class_getitem__ method (which origin is one the protocol defined in
+        # collections module) whereas the typing module consider it should not.
```

Comment:
```suggestion
        # If we are here, then we are sure to modify an object that does have
        # __class_getitem__ method (which origin is the protocol defined in
        # collections module) whereas the typing module considers it should not.
```

I think? I really don't understand the sentence after `(which ...`